### PR TITLE
Create campaign

### PR DESCRIPTION
**Pull request by tamlyn10**

### CODE REVIEW

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -13,7 +13,7 @@
     <header class="container-fluid">
       <nav class="row">
         <div class="col-sm-4"></div>
-        <h1 class="col-sm-4 text-center">Barefoot <span class="highlight-color">Bidders</span></h1>
+        <a href="/"><h1 class="col-sm-4 text-center">Barefoot <span class="highlight-color">Bidders</span></h1></a>
```
Comment (DannySantos) [inline app/views/layouts/application.html.erb]: This should use a helper:

<%= link_to root_path do %>
  <h1>Barefoot <span class="highlight-color">Bidders</span></h1>
<% end %>

With this setup I'd also wrap the whole thing in a div that has the Bootstrap grid classes

File: app/assets/stylesheets/OneFrameworkToRuleThemAll/one-forms.scss [hunk 2]
```diff
@@ -25,4 +27,22 @@
   background-image: none;
   border: 1px solid transparent;
   border-radius: 4px;
+}
+
+.btn-wide {
```
Comment (DannySantos) [inline app/assets/stylesheets/OneFrameworkToRuleThemAll/one-forms.scss]: You don't need to repeat yourself here, give the link a class of btn AND btn-wide and then you can delete from btn-wide all of the stuff that has been repeated from btn

File: app/assets/stylesheets/new_campaign.scss [hunk 3]
```diff
@@ -0,0 +1,14 @@
+.campaign-form-label {
```
Comment (DannySantos) [inline app/assets/stylesheets/new_campaign.scss]: There's no need for this to be a specific CSS class. You should wrap the campaign form content in a <main> tag with a campaign or new-campaign class name and then you can just do something like:

.campaign-new {
  label {

  }
}

File: app/controllers/campaigns_controller.rb [hunk 4]
```diff
@@ -5,5 +5,20 @@ def show
...
+  def create
+    campaign = Campaign.create(campaign_params)
+    redirect_to user_campaign_path(current_user, campaign)
+  end
+  
+  private
+  
+  def campaign_params
+    user = params.permit(:user_id)
+    params.require(:campaign).permit(:question, :option_1, :option_2, :user_id).merge(user)
```
Comment (DannySantos) [inline app/controllers/campaigns_controller.rb]: Remind me what merge does in this context? Just curiosity

File: app/assets/stylesheets/OneFrameworkToRuleThemAll/one-forms.scss (hunk 2 above)
Comment (tamlyn10) [inline app/assets/stylesheets/OneFrameworkToRuleThemAll/one-forms.scss]: cool... thanks

File: features/campaign.feature [hunk 5]
```diff
@@ -11,4 +11,14 @@ Scenario: A visitor views a campaign
   Given there are some users
     And one of them has listed a campaign
   When they visit the campaign
-  Then they will see the campaign page
\ No newline at end of file
+  Then they will see the campaign page
+  
+@wip
```
Comment (DannySantos) [inline features/campaign.feature]: Lose the wip Indiana Jones :)

File: app/assets/stylesheets/new_campaign.scss (hunk 3 above)
Comment (tamlyn10) [inline app/assets/stylesheets/new_campaign.scss]: ok

**Pull request merged**